README.md

```diff
@@ -76,6 +76,72 @@ extension MyCustomViewController {
 }
 ```
 
+###Reusable Views To Improve Performance
+
+Collections and tables view use `reuseidentifier` on cell to recycle a view
+
+Each custom collection and table view controller is extended with this code and provide a list of available reuse identifier
+
+`Reusable` enumeration contains list of available reusable identifier
+
+`kind` property represent types of view (table, collection)
+
+`viewType` property return type of view if specified in storyboard.
+
+```swift
+extension MyCustomTableViewController { 
```

Comment:
this is "implementation detail" no need to put it to README. Example of usage is enough.